ibis/backends/base/sql/registry/window.py

```diff
@@ -95,7 +95,10 @@ def cumulative_to_window(translator, expr, window):
 
     klass = _cumulative_to_reduction[type(op)]
     new_op = klass(*op.args)
-    new_expr = expr._factory(new_op, name=expr._name)
+    new_expr = new_op.to_expr()
+
+    if (name := expr._safe_name) is not None:
```

Comment:
Oooo, we can now use walrus?

Reply:
Yep, it was introduced in 3.8 and we're >= 3.8 :)